agent/engine/docker_task_engine.go

```diff
@@ -1016,7 +1016,8 @@ func (engine *DockerTaskEngine) createContainer(task *apitask.Task, container *a
 	// Create metadata directory and file then populate it with common metadata of all containers of this task
 	// Afterwards add this directory to the container's mounts if file creation was successful
 	if engine.cfg.ContainerMetadataEnabled && !container.IsInternal() {
-		mderr := engine.metadataManager.Create(config, hostConfig, task, container.Name)
+		info, _ := engine.Info()
```

Comment:
Should probably handle the error returned from `Info()` function.

Reply:
Done. On Info() error, warning is sent and security options will be initialized to empty array.